drivers/virtualbox/virtualbox.go

```diff
@@ -452,19 +452,19 @@ func (d *Driver) GetIP() (string, error) {
 	if s != state.Running {
 		return "", drivers.ErrHostIsNotRunning
 	}
-	cmd, err := drivers.GetSSHCommandFromDriver(d, "ip addr show dev eth1")
+	output, err := drivers.RunSSHCommandFromDriver(d, "ip addr show dev eth1")
 	if err != nil {
 		return "", err
 	}
 
-	// reset to nil as if using from Host Stdout is already set when using DEBUG
-	cmd.Stdout = nil
+	var buf bytes.Buffer
+	buf.ReadFrom(output.Stdout)
```

Comment:
is you intention here to see if you can read something from `output.Stdout`, if so you can probably do:

```
nn, err := io.Read([]byte{1})
// check err
// if nn == 0 { compose some error here. }
```